hamocc/mo_param_bgc.F90

```diff
@@ -162,12 +173,13 @@ module mo_param_bgc
   real, protected :: tf0        = -2.7819
   real, protected :: tff        =  0.2395
 
-  ! Initial fractionation during photosynthesis
-  real :: bifr13 = 0.98
-  real :: bifr14
-
-  ! Decay parameter for sco214, HalfLive = 5730 years
-  real, parameter :: c14_t_half = 5700.*365.      ! Half life of 14C [days]
+  ! Set constants for dms scheme following Kloster et al. (2006), Table 1
```

Comment:
Stupid question: any reason to keep the reverse order of initialization?

Reply:
No, just laziness - will fix this.